aiokafka/client.py

```diff
@@ -292,7 +292,7 @@ def get_random_node(self):
 
             try:
                 metadata = await conn.send(metadata_request)
-            except KafkaError as err:
+            except (KafkaError, asyncio.TimeoutError) as err:
```

Comment:
I kind of wander why I did not use `client.send` here. That would handle the issue correctly... Probably we should also close the connection here as we do in send, but that is another enhancement ))